lib/util/git/index.ts

```diff
@@ -875,6 +877,16 @@ export async function prepareCommit({
           let contents: Buffer;
           // istanbul ignore else
           if (typeof file.contents === 'string') {
+            if (/migrate-config/g.exec(branchName)) {
+              const prettierConfigPath = readLocalPath(fileName);
+              const options = await prettier.resolveConfig(prettierConfigPath);
```

Comment:
we shouldn't do it. it would allow running JavaScript from prettier config. just use default settings for now.

maybe we can use our existing indent detection   if we need that information.

Reply:
I also don't think this is good to do at the git level